src/zenml/new/pipelines/pipeline.py

```diff
@@ -691,6 +693,33 @@ def _run(
 
             self.log_pipeline_deployment_metadata(deployment_model)
 
+            # TODO: We should probably only do this if there is no schedule?
```

Comment:
100%, when running a pipeline with schedule we already create the schedule object from which runs will then be created at the corresponding scheduling times - don't think it makes sense to create a placeholder for the first run of a schedule from a UX perspective

Reply:
Currently, we create this schedule object even if the orchestrator does not support scheduling. Do you think we should do the same in this case, only create the placeholder run if the user does not pass a schedule.

That means if they pass a schedule but the orchestrator does not support scheduling, it'll still work but they won't be able to see it in the dashboard immediately.

The alternative would be to add a property `supports_scheduling` on the orchestrator I guess, and do it only if a schedule is passed and the orchestrator supports it?